packages/plugins/rubygem-smart_proxy_m2/rubygem-smart_proxy_m2.spec

```diff
@@ -0,0 +1,92 @@
+%global gem_name smart_proxy_m2
+%global plugin_name m2
+
+%global foreman_proxy_min_version 1.20
+%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+Name: rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: M2 smart proxy plugin for Foreman
+Group: Applications/Internet
+License: GPLv3
+URL: http://github.com/ianballou/smart_proxy_m2
```

Comment:
Could you use https here? I've submitted https://github.com/ianballou/smart_proxy_m2/pull/1 to correct the gemspec as well. Note that the License in the RPM spec is in the **correct** form now and doesn't need updating.